src/model/FeaturesView.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { Fragment, useCallback } from 'react';
+import ProtvistaTrack from 'protvista-track';
+import ProtvistaManager from 'protvista-manager';
+import ProtvistaDatatable from 'protvista-datatable';
+import ProtvistaSequence from 'protvista-sequence';
+import ProtvistaNavigation from 'protvista-navigation';
+import { loadWebComponent } from '../utils/utils';
+
+type FeatureLocation = {
+  value: number;
+  modifier: string;
+};
+
+type Feature = {
+  type: string;
+  featureId: string;
+  description: string;
+  location: {
+    start: FeatureLocation;
+    end: FeatureLocation;
+  };
+};
+
+type ProtvistaFeature = {
+  type: string;
```

Comment:
Can `ProtvistaFeature.type` become an enum?

Reply:
The type is coming from the API and there is always scope for new feature types being added/changed, so I would rather not.